modules/nodes-pods-autoscaling-custom-creating-workload.adoc

```diff
@@ -97,13 +97,13 @@ spec:
         omitStages: "RequestReceived" 
         omitManagedFields: false 
       lifetime: 
-        maxAge: "2"
-        maxBackup: "1"
-        maxSize: "50"
+        maxAge: '2'
+        maxBackup: '1'
+        maxSize: '50'
   fallback: <11>
-    failureThreshold: 3
-    replicas: 6
-  pollingInterval: 30 <12>
+    failureThreshold: '3'
+    replicas: '6'
+  pollingInterval: '30' <12>
```

Comment:
please revert this, the quotoes are only needed for values in `spec.triggers.metadata`